.github/ISSUE_TEMPLATE/update-airflow.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Update Airflow
+about: >-
+  This template contains instructions specific to updating this product and/or
+  container image(s).
+title: >-
+  chore(airflow): Update container images ahead of Stackable Release XX.(X)X
+labels: []
+projects: ['stackabletech/10']
+assignees: ''
+---
+
+Part of #xxx.
+
+> [!TIP]
+> Please add the `scheduled-for/20XX-XX` label.
+
+```[tasklist]
+### Update tasks
+- [ ] Update `versions.py` to reflect the agreed upon versions in the spreadsheet (including the removal of old versions).
+- [ ] Download new constraints file (see `airflow/download_constraints.sh`).
+- [ ] Update other dependencies if applicable (eg: python, statsd_exporter, etc).
+- [ ] Check other operators (getting_started / kuttl) for usage of the versions. Add the PR(s) to the list below.
+- [ ] Update the version in demos. Add the PR(s) to the list below.
+```
+
+```[tasklist]
+### Related Pull Requests
+- [ ] _Link to the docker-images PR (product update)_
+- [ ] _Link to operator PR (getting_started / kuttl)_
+- [ ] _Link to any other operator PRs (getting_started / kuttl)_
+- [ ] _Link to demo PR (raise against the `next` branch)_
+```
+
+<!--
+Make this a regular list so it isn't easily editable from the rendered
+description?
+-->
+```[tasklist]
+### Acceptance
```

Comment:
Nice to have: confirmation that it works on Openshift, too (although that could be arguably covered by the operator PR where we already have a "works on Openshift" check).

Reply:
I should have mentioned it, but I specifically left out comprehensive testing from these task lists (when it was a PR template). But now that it is a tracking issue, I could add that acceptance criteria back in.

~The instruction then would be to run OpenShift tests in Jenkins (and some instructions at the end for steps to doing that), right?~

---

> although that could be arguably covered by the operator PR where we already have a "works on Openshift" check

Yeah, I guess it is covered somewhere. Thinking more about it now, the goal for the product updates is to add in the new versions (whether or not they work - the containers are built and available). So I do feel the testing for that should be covered by the respective operator.

Reply:
Yes, fair enough. Testing on Openshift with replicated should now not be any more involved than other platforms.